### DISCUSSION TITLE
Why is 'owner' a mandatory argument for tasks and dags?

### DISCUSSION THREAD

**Discussion by caitpj** [apache/airflow#35734] [Q&A] [answered]
This seems like a strange argument to make mandatory, since it is just documentation of sorts.

**Comment by potiuk** [comment]
What makes you think it's mandatory ?

**Reply by caitpj** [reply]
It's in the [fundamentals tutorial](https://airflow.apache.org/docs/apache-airflow/stable/tutorial/fundamentals.html):

> A task must include or inherit the arguments task_id and owner, otherwise Airflow will raise an exception.

I tried playing around on my local airflow to remove the default_owner value in the config.yml file, but, as the the above indicates, I get an exception. 

I see in airflow/models/abstractoperator.py line 58:
 `DEFAULT_OWNER: str = conf.get_mandatory_value("operators", "default_owner")`

**Reply by potiuk** [reply, marked as answer]
Why do you want to remove or change the behaviour ? 

Since it is set by default, it's not **really**  mandatory in the sense it's not needed to be specified. It would be great if you submit PR explaining that in the "fundamentals" page. Just click on the "Suggest a change on this page" button and submit PR that will be opened for you. 

No idea why it is set (It's probably been there for 10 years like that - since the beginning and unless there is a good reason to change it, we should probably keep it - for backwards compatibility - some people might rely on it being present.

**Reply by caitpj** [reply]
Thanks for the detailed response, @potiuk. I have done as you suggest, [here](https://github.com/apache/airflow/pull/35740) is the PR.
